docs/overview.rst

```diff
@@ -102,3 +102,65 @@ For example, the `Escrow` contract factory requires linking to an instance of th
        deployer.register_strategy("Escrow", escrow_strategy)
        linked_escrow_package, _ = deployer.deploy("Escrow") 
        return linked_escrow_package.deployments.get_deployment("Escrow")
+
+
+Log
+---
+
+The ``Log`` class is available to help with testing for contract events, and the contents of the emitted logs.
+
+.. code:: python
+
+   # SETUP
+   ping_contract = """
+   Ping: event({_first: indexed(bytes32), _second: bytes32})
+
+   @public
+   def __init__():
+       pass
+
+   @public
+   def ping(first: bytes32, second: bytes32):
+	   log.Ping(first, second)
+   """
+   ping_deployer = vy_deployer.deploy("ping")
+   ping_instance = ping_deployer.deployments.get_contract_instance("ping")
+   tx_hash = ping.functions.ping(b"one", b"two")
+   receipt = w3.eth.waitForTransactionReceipt(tx_hash)
+
+.. py:method:: Log(contract_event, *args).is_present(receipt)
```

Comment:
If you used autodoc here, you could embed this documentation into the function docstring itself which I think is a more ideal way for us to organize our documentation.